.github/ISSUE_TEMPLATE.md

```diff
@@ -1,34 +1,45 @@
 <!-- To file a bug, fill out the information below. To file something else, like a suggestion or issue for discussion, delete the below and write what you want. -->
 
 <!-- If they're relevant, images that show what happens when you trigger the bug. -->
+
 ### Images:
 
 <!-- What steps can we take to recreate the bug? -->
+
 ### Replicate:
-1. 
-2. 
-3. 
+
+1.  <!-- Step 1 to Replicate the bug -->
```

Comment:
Not sure why this didn't show up before, but I'll put it in again:

This isn't really fixed. People will have to know to do this whenever they do a list in markdown.

In some future social experiment, I'd also be interested to see if people who don't know HTML and are filling in the template will get confused about where they're supposed to put their text...